tables/elb_access_log/elb_access_log.go

```diff
@@ -0,0 +1,214 @@
+package elb_access_log
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type ElbAccessLogBatch struct {
+	Records []ElbAccessLog `json:"Records"`
+}
+
+type ElbAccessLog struct {
+	schema.CommonFields
+
+	ActionsExecuted        string    `json:"actions_executed"`
```

Comment:
Which fields can be empty (or `-`)? Do we have `omitempty` on all of them?

Reply:
Yes, added `omitempty` to all of them